### PR TITLE
v3.5.0 – Adding note and close-circle icons

### PR DESCRIPTION
**Pull request by Parkyster**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
+v3.5.0
+------------------------------
+*March 24, 2021*
 
+### Added
+- `note` icon added.
+- `close-circle` icon added.
```
Comment (benwhite-deltas) [inline CHANGELOG.md:11]: Please keep the spacing:
```suggestion


v3.5.0
------------------------------
*March 24, 2021*

### Added
- `note` icon added.
- `close-circle` icon added.

```

File: src/img/icons/general/close-circle.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24"><defs><path id="a" d="M4.927 4.927c3.905-3.905 10.236-3.905 14.141 0a10 10 0 11-14.141 0zm11.518.422a8.002 8.002 0 00-10.102.994A7.948 7.948 0 004 12a8 8 0 1012.445-6.651zM15.197 7.8c.221 0 .435.073.609.207l.099.087a1.002 1.002 0 010 1.417L13.417 12l2.488 2.488a1.003 1.003 0 01.087 1.318l-.087.099a1.002 1.002 0 01-1.417 0L12 13.417l-2.488 2.488a1.003 1.003 0 01-1.318.087l-.099-.087a1.002 1.002 0 010-1.417L10.583 12 8.097 9.512a.999.999 0 01-.087-1.318l.087-.099a.999.999 0 011.416 0L12 10.583l2.488-2.488c.188-.188.443-.294.709-.294z"/></defs><use fill="#D50525" xlink:href="#a" fill-rule="evenodd"/></svg>
```
Comment (benwhite-deltas) [inline src/img/icons/general/close-circle.svg:1]: Have you run this through SVGOMG? I'm not sure the xlink is needed - I think the path could be defined inline, and I'd expect SVGOMG to fix this.

Comment (Parkyster) [reply src/img/icons/general/close-circle.svg:1]: @benwhite-deltas I have with the default options

**Review by benwhite-deltas — APPROVED** [review]

**Comment by dandel10n** [comment]
Note.svg looks like SortIcon.svg that we already have in here. You can scroll through the list of icons [here](https://justeat.github.io/fozzie-components/@justeat/storybook/index.html?path=/story/components-atoms--icons)

**Comment by Parkyster** [comment]
> Note.svg looks like sort.svg that we already have in here. You can scroll through the list of icons [here](https://justeat.github.io/fozzie-components/@justeat/storybook/index.html?path=/story/components-atoms--icons)

Its similar but not quite the same if you look at the lengths of the bars and also the line spacing.

File: src/img/icons/general/close-circle.svg (hunk 2 above)
Comment (DamianMullins) [inline src/img/icons/general/close-circle.svg:1]: Can you run the original SVG through SVGOMG again (sorry!) add ensure that you have the "Clean IDs" setting turned off? Also, we ten to leave the "pretty markup" setting on as it makes the file contents easier to scan and it all gets minified further down the line anyway.

Either that or manually update the "#a" id to include the filename as a prefix e.g. `#close-circle-a". I had to manually do this to a bunch of SVGs recently as the ids can conflict and cause visual issues as well as invalidate the markup.

Comment (dandel10n) [reply src/img/icons/general/close-circle.svg:1]: I will add this to the readme as we have these comments in every PR

Comment (dandel10n) [reply src/img/icons/general/close-circle.svg:1]: Another thing is to choose "Prefer viewBox to width/height"

Comment (DamianMullins) [reply src/img/icons/general/close-circle.svg:1]: Agree, we should define the recommended properties when using SVGOMG.

**Review by dandel10n — APPROVED** [review]

**Review by 1itvinka — APPROVED** [review]

**Pull request merged**